src/components/Button.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function Button({children, buttonHandler, className, onClick}) {
+
+    return (
+        <>
+            <button  className={className} onClick={onClick}>{children}</button>
+        </>
+    )
+
+
+}
```

Comment:
If the component returns a single line of html you could use `<button .. />` directly without an extra component wrapped around it.